storage/datastore/store.go

```diff
@@ -554,6 +563,111 @@ func (d *Datastore) DeleteAccount(id model.ProjectChildID) error {
 	return txErr
 }
 
+// Contracts
+
+func (d *Datastore) InsertContract(con *model.Contract) error {
+	ctx, cancel := context.WithTimeout(context.Background(), d.conf.DatastoreTimeout)
+	defer cancel()
+
+	_, txErr := d.dsClient.RunInTransaction(ctx, func(tx *datastore.Transaction) error {
+
+		proj := &model.InternalProject{
+			ID: con.ProjectID,
+		}
+
+		err := tx.Get(proj.NameKey(), proj)
+		if err != nil {
+			return err
+		}
+
+		//con.Index = proj.TransactionTemplateCount
+
+		//proj.ContractCount++
```

Comment:
Do we need those?

Reply:
this is the reference code as i referred from TranscactionTemplate struct. I will remove it along with other comments prefixed with //soe

Index in Contract struct is used to track index on the account it belongs to.

I wrote a documentation here: https://www.notion.so/Notes-for-changes-Flipfest-21-b2c49263e87a4551b57826070fe566a6?showMoveTo=true